src/qt/forms/createwalletdialog.ui

```diff
@@ -17,6 +17,41 @@
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item alignment="Qt::AlignHCenter">
+    <widget class="QLabel" name="label">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="text">
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You are one step away from creating your new descriptors wallet!&lt;/p&gt;&lt;p&gt;Please enter the name and, if desired, enable any custom options&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
HTML markups will go as a part translatable strings. Can it be avoided?

Is it better to replace "custom options" with "advanced options" as they are named in https://github.com/bitcoin-core/gui/blob/6b83663eb8ed06216351e76439b825de4840a481/src/qt/forms/createwalletdialog.ui#L111 ?

Reply:
And "new descriptors wallet" --> "new descriptor wallet" probably?

Reply:
pushed.